content/en/api/v1/rate-limits/_index.md

```diff
@@ -25,6 +29,168 @@ Regarding the API rate limit policy:
 | `X-RateLimit-Reset`     | time in seconds until next reset.                        |
 | `X-RateLimit-Name`      | name of the rate limit for increase requests             |
 
+### Datadog API Usage Metrics
```

Comment:
I didn't duplicate the suggestions from above, but let me know if I can help with making sure everything's consistent 🙂

Reply:
yeah can copy paste these from latest example